src/librustc_resolve/lib.rs

```diff
@@ -2194,7 +2177,9 @@ impl<'a, 'tcx> Resolver<'a, 'tcx> {
     // user and one 'x' came from the macro.
     fn binding_mode_map(&mut self, pat: &Pat) -> BindingMap {
         let mut result = HashMap::new();
-        pat_bindings(&self.def_map, pat, |binding_mode, _id, sp, path1| {
+        pat_bindings(&self.def_map,
+                     pat,
+                     |binding_mode, _id, sp, path1| {
```

Comment:
You effectively read it as a line from pat_bindings( until the `{`. If you treat that as the function arguments, it's not longer than the text width here. I think that would work better.